log.go

```
@@ -50,21 +50,29 @@ func (a AntiLog) Write(msg string, fields ...Field) {
 		PrependUnique(encodeFieldList(fields)).
 		PrependUnique(a.Fields)
 
-	for ix := 0; ix < len(encodedFields); ix++ {
-		field := encodedFields[ix]
-		if field.Key() != "message" || field.Key() != "timestamp" {
+	var sb bytes.Buffer
+	sb.WriteString(`{ "timestamp": "`)
+	sb.WriteString(now.Format(time.RFC3339))
+	sb.WriteString(`", "message": `)
+	json.NewEncoder(&sb).Encode(msg)
```

As below - should probably explicitly ignore the error here 
```
_ = json.New...
```